hangups/http_utils.py

```diff
@@ -30,9 +30,9 @@ def fetch(method, url, params=None, headers=None, cookies=None, data=None,
     error_msg = None
     for retry_num in range(MAX_RETRIES):
         try:
-            res = yield from asyncio.wait_for(aiohttp.request(
-                method, url, params=params, headers=headers, cookies=cookies,
-                data=data, connector=connector
+            res = yield from asyncio.wait_for(session.request(
+                method, url, params=params, headers=headers, data=data,
+                proxy=os.environ.get('HTTP_PROXY')
```

Comment:
We can live with that for now. The HTTP request is going to take thousands of microseconds anyways. :smile: